package.json

```diff
@@ -4,20 +4,11 @@
   "description": "Simple editor to create responsive content",
   "main": "./editor.js",
   "dependencies": {
-    "browserify": "^9.0.3",
```

Comment:
Actually, all these dependencies are not required in production.
They are included in editor.js
So I propose move them all to dev.